.github/workflows/go.yml

```diff
@@ -20,3 +20,17 @@ jobs:
         run: make bin
       - name: Run tests
         run: make test
+
+  check-tidy:
+    name: Check tidy
+    runs-on: [ubuntu-latest]
+    steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+      - name: Check-out code
+        uses: actions/checkout@v2
+      # tidy check need to be run before code generation which will regenerate codes.
```

Comment:
A minor question, why tidy check needs run before code generation?

Reply:
Bad copy paste accident (as all of the above)...